antsibull/lint_extra_docs.py

```diff
@@ -40,6 +42,22 @@ def lint_optional_conditions(content: str, path: str, collection_name: str
 
     Return a list of errors.
     '''
+
+    # Make sure Pygments knows about the additional lexers
+    try:
+        from pygments.lexers import LEXERS
+    except ImportError:
+        pass
+    else:
+        for lexer_name, lexer_class in [
+            ('AnsibleOutputLexer', AnsibleOutputLexer)
+        ]:
+            LEXERS[lexer_name] = (
```

Comment:
I wonder if just declaring an entrypoint will do the same in a cleaner manner.

Reply:
It was a bit tricky to find out how to do that with poetry. The documentation on that part of poetry is IMO hard to understand (https://python-poetry.org/docs/pyproject/#plugins, see also https://github.com/python-poetry/poetry/issues/927 and https://github.com/python-poetry/poetry/issues/658). Anyway, now it works :)